Google Cloud/Google Kubernetes Engine/ingest/parser.yml

```
@@ -0,0 +1,134 @@
+  - name: parse_jsonPayload
+  - name: parse_involvedObject
+  - name: parse_metadata
+  - name: parse_resource_labels
+  - name: parse_resource
+  - name: parse_source
+  - name: parse_kv_data
+
+stages:
+
+  google_kubernetes_engine:
+    actions:
+      - set:
+          event.kind: "event"
+          event.category: "process"
+          event.type: ['change']
+
+          '@timestamp': "{{json_event.message.timestamp}}"
+
+
+          google_kubernetes_engine.insertId: "{{json_event.message.insertId}}"
+          google_kubernetes_engine.logName: "{{json_event.message.logName}}"
+          google_kubernetes_engine.receiveTimestamp: "{{json_event.message.receiveTimestamp}}"
+          google_kubernetes_engine.severity: "{{json_event.message.severity}}"
+
+  parse_kv_data:
+    actions:
+      - set:
+          # This field does not work
+          event.reason: "{{parse_kv.message.msg}}"
+    filter: "{{parse_kv.message != null}}"
+
+  parse_jsonPayload:
+    actions:
+      - set:
+
+          event.reason: "{{json_event.message.jsonPayload.MESSAGE }}"
+          google_kubernetes_engine.jsonPayload.apiVersion: "{{json_event.message.jsonPayload.apiVersion}}"
+          google_kubernetes_engine.jsonPayload.eventTime: "{{json_event.message.jsonPayload.eventTime}}"
+          google_kubernetes_engine.jsonPayload.kind: "{{json_event.message.jsonPayload.kind}}"
+          google_kubernetes_engine.jsonPayload.message: "{{json_event.message.jsonPayload.message}}"
+          google_kubernetes_engine.jsonPayload.reason: "{{json_event.message.jsonPayload.reason}}"
+          google_kubernetes_engine.jsonPayload.reportingComponent: "{{json_event.message.jsonPayload.reportingComponent}}"
+          google_kubernetes_engine.jsonPayload.reportingInstance: "{{json_event.message.jsonPayload.reportingInstance}}"
+          google_kubernetes_engine.jsonPayload.type: "{{json_event.message.jsonPayload.type}}"
+          google_kubernetes_engine.jsonPayload.MESSAGE: "{{json_event.message.jsonPayload.MESSAGE}}"
+          log.syslog.priority: "{{json_event.message.jsonPayload.PRIORITY}}"
+          log.syslog.facility.code: "{{json_event.message.jsonPayload.SYSLOG_FACILITY}}"
+          # Should be replaced by log.syslog.msgid when available
+          google_kubernetes_engine.jsonPayload.SYSLOG_IDENTIFIER: "{{json_event.message.jsonPayload.SYSLOG_IDENTIFIER}}"
+          # Should be replaced by host.boot.id when available
+          google_kubernetes_engine.jsonPayload._BOOT_ID: "{{json_event.message.jsonPayload._BOOT_ID}}"
+          google_kubernetes_engine.jsonPayload._CAP_EFFECTIVE: "{{json_event.message.jsonPayload._CAP_EFFECTIVE}}"
+          process.command_line: "{{json_event.message.jsonPayload._CMDLINE}}"
+          google_kubernetes_engine.jsonPayload._COMM: "{{json_event.message.jsonPayload._COMM}}"
+          process.executable: "{{json_event.message.jsonPayload._EXE}}"
+          file.gid: "{{json_event.message.jsonPayload._GID}}"
```

Maybe use [`process.group.id`](https://www.elastic.co/guide/en/ecs/8.2/ecs-group.html#field-group-id) (even if the field is in beta)
```suggestion
          process.group.id: "{{json_event.message.jsonPayload._GID}}"
```